web/beacon-app/src/features/topics/hooks/useFetchTopic.ts

```diff
@@ -0,0 +1,22 @@
+import { useQuery } from '@tanstack/react-query';
+
+import axiosInstance from '@/application/api/ApiService';
+import { RQK } from '@/constants/queryKeys';
+
+import { topicRequest } from '../api/topicDetailApiService';
+import { TopicQuery } from '../types/topicService';
+
+export function useFetchTopics(topicsID: string): TopicQuery {
```

Comment:
```suggestion
export function useFetchTopic(topicsID: string): TopicQuery {
```

Reply:
If this is for one topic, then should the name be updated so that there isn't any confusion with `useFetchTopics` for multiple topics?